main.go

```diff
@@ -144,20 +140,20 @@ func main() {
 
 	// Connect to database
 	var err error
-	logbuch.Info("starting with %s database", config.Db.Dialect)
+	slog.Info("starting with database", "dialect", config.Db.Dialect)
 	db, err = gorm.Open(config.Db.GetDialector(), &gorm.Config{Logger: gormLogger}, conf.GetWakapiDBOpts(&config.Db))
 	if err != nil {
-		logbuch.Error(err.Error())
-		logbuch.Fatal("could not open database")
+		slog.Error("could not connect to database", "error", err)
+		log.Fatal("could not open database")
```

Comment:
Or we only need `log.Fatal` since `log.Fatal` also structure, but the problem is this method still use the INFO level
<img width="1332" alt="image" src="https://github.com/user-attachments/assets/649034b0-cb51-430f-845b-db0e9006da28">
The options here
1. We just use the log.Fatal for now
2. We write a shared function `logFatal(...interface{})` that wrap `slog.Error` and `os.Exit()`
What do you think?

Reply:
I updated the code to only use log.Fatal in the mean time

Reply:
Not entirely happy about this. Would prefer to have errors logged to `stderr` consistently. However, this is only an interim solution until we switch to the `slog-sentry` package anyway, isn't it?

Reply:
> However, this is only an interim solution until we switch to the slog-sentry package anyway, isn't it?

We will revisit this when we switch to `slog-sentry`.

For this PR. I realize we don't need another wrapper to cover the missing of `slog.Fatal`. We can use `Log().Fatal()`, it already wrapped the `slog.Error` and `os.Exit(1)`. I pushed a commit to replace all `log.Fatal` by the `Log().Fatal()`